src/components/InstitutionAdminDashboard/InstitutionAdminDashboard.tsx

```diff
@@ -72,6 +75,17 @@ const InstitutionAdminDashboard: FC<InstitutionAdminDashboardProps> = ({ type, p
     },
   });
 
+  const openNotification = (): void => {
```

Comment:
Any chance you can make this global and pass any required params like styles etc? Might even be able to do as a component with its own styles file? It's being used in a few places now